robottelo/ui/sync.py

```diff
@@ -172,10 +172,7 @@ def enable_rh_repos(self, repos_tree):
                     repo_name = repo_values['repo_name']
                     # UI is very slow here. Hence timeout is 120 seconds.
                     self.click(
-                        (strategy3, value3 % repo_name),
-                        waiter_timeout=120,
-                        scroll=True,
-                    )
+                        (strategy3, value3 % repo_name), waiter_timeout=120)
```

Comment:
Yes :+1: to @omaciel's comments.  Lets not change this code and lets fix the main `click` method so if we need to change the `click` method in future for newer versions of selenium, the sync plan change here will be intact.